From f4743f5d3719abd43023f52292760b7efc96b474 Mon Sep 17 00:00:00 2001
Message-Id: <f4743f5d3719abd43023f52292760b7efc96b474.1350493760.git.minovotn@redhat.com>
In-Reply-To: <c93188b6c6973932d2adaa52e6a4920db13b4e62.1350493760.git.minovotn@redhat.com>
References: <c93188b6c6973932d2adaa52e6a4920db13b4e62.1350493760.git.minovotn@redhat.com>
From: Jeffrey Cody <jcody@redhat.com>
Date: Wed, 17 Oct 2012 05:59:37 +0200
Subject: [PATCH 24/35] block: vpc image file reopen

RH-Author: Jeffrey Cody <jcody@redhat.com>
Message-id: <a6161d0c00d1c7c9356bf4aae78580c8a1040065.1350447475.git.jcody@redhat.com>
Patchwork-id: 43280
O-Subject: [RHEL6.4 qemu-kvm PATCH v4 24/35] block: vpc image file reopen
Bugzilla: 767233
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Eric Blake <eblake@redhat.com>
RH-Acked-by: Kevin Wolf <kwolf@redhat.com>

There is currently nothing that needs to be done for VPC image
file reopen.

Signed-off-by: Jeff Cody <jcody@redhat.com>
Signed-off-by: Kevin Wolf <kwolf@redhat.com>
(cherry picked from commit 3fe4b70008f3a0323e1d685becc6a9cff2b71de7)
---
 block/vpc.c | 7 +++++++
 1 file changed, 7 insertions(+)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 block/vpc.c | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/block/vpc.c b/block/vpc.c
index d11fe34..9e6bf83 100644
--- a/block/vpc.c
+++ b/block/vpc.c
@@ -227,6 +227,12 @@ static int vpc_open(BlockDriverState *bs, int flags)
     return -1;
 }
 
+static int vpc_reopen_prepare(BDRVReopenState *state,
+                              BlockReopenQueue *queue, Error **errp)
+{
+    return 0;
+}
+
 /*
  * Returns the absolute byte offset of the given sector in the image file.
  * If the sector is not allocated, -1 is returned instead.
@@ -629,6 +635,7 @@ static BlockDriver bdrv_vpc = {
     .bdrv_write     = vpc_co_write,
     .bdrv_co_flush  = vpc_co_flush,
     .bdrv_close     = vpc_close,
+    .bdrv_reopen_prepare = vpc_reopen_prepare,
     .bdrv_create    = vpc_create,
 
     .create_options = vpc_create_options,
-- 
1.7.11.7

